src/components/FAQSection.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Accordion, Card } from 'react-bootstrap';
+
+const styles = {
+  Section: {
+    margin: '1em 0'
+  },
+  Accordion: {
+    margin: '1em 0'
+  },
+  Question: {
+    color: 'black',
+    cursor: 'pointer'
+  },
+  Answer: {
+    color: 'grey',
```

Comment:
there's a specific css var for a grey `var(--main-grey)` not sure if it's the same as what you're looking for